test/integration/consensus/consensus_sunny_day.cpp

```diff
@@ -29,10 +29,9 @@ using namespace iroha::model;
 using namespace iroha::consensus::yac;
 using namespace framework::test_subscriber;
 
-Peer mk_local_peer(uint64_t num) {
-  Peer peer;
-  peer.address = "0.0.0.0:" + std::to_string(num);
-  return peer;
+auto mk_local_peer(uint64_t num) {
+  auto address = "0.0.0.0:" + std::to_string(num);
```

Comment:
D: why "0.0.0.0" is considered a local peer
Replace with 127.0.0.1 plz, also I think you may inline this var